src/ui/screens/SendFundScreen.tsx

```diff
@@ -133,6 +172,15 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     borderRadius: 10,
   },
+  keepAlive: {
+    flexDirection: 'row',
+    justifyContent: 'flex-end',
+    paddingVertical: 15,
```

Comment:
NIT 💅 

We could use the standard padding or a multiple of that.

```suggestion
    paddingVertical: standardPadding * 2,
```